@kiva/kv-components/vue/KvCartModal.vue

```diff
@@ -0,0 +1,359 @@
+<template>
+	<transition
+		enter-active-class="tw-transition-opacity tw-duration-300"
+		leave-active-class="tw-transition-opacity tw-duration-300"
+		enter-class="tw-opacity-0"
+		enter-to-class="tw-opacity-full"
+		leave-class="tw-opacity-full"
+		leave-to-class="tw-opacity-0"
+	>
+		<!-- the screen -->
+		<div
+			v-show="visible"
+			class="
+				tw-z-modal
+				tw-absolute
+				tw-inset-0
+				tw-bg-black
+				tw-bg-opacity-[75%]
```

Comment:
This JIT tailwind class should be moved to a custom class, since we don't want to populate the compiled CSS file with a ton of JIT classes.